tests/testthat/test-plot-grid.R

```diff
@@ -0,0 +1,53 @@
+test_that("plots grid is rendered correctly", {
+  skip_if_not_installed("vdiffr")
+  skip_if_not_installed("tlf")
+  skip_if(getRversion() < "4.1")
+
+  library(tlf)
+
+  set.seed(123)
+  ls_plots <- list(
+    tlf::plotHistogram(x = rnorm(100)),
+    tlf::plotHistogram(x = rnorm(100, mean = 3)),
+    tlf::plotHistogram(x = rnorm(100, mean = 10))
+  )
+
+  plotGridObj <- PlotGridConfiguration$new(ls_plots)
+
+  plotGridObj$title <- "my combined plot"
+  plotGridObj$subtitle <- "something clever"
+  plotGridObj$caption <- "something dumb"
+  plotGridObj$nColumns <- 2L
+  plotGridObj$tagLevels <- "A"
+  plotGridObj$tagPrefix <- "Plot ("
+  plotGridObj$tagSuffix <- ")"
+
+  set.seed(123)
+  vdiffr::expect_doppelganger(
```

Comment:
What does this do?

Reply:
Roughly speaking, it checks if the generated image matches the existing SVG representing the expected plot.

Reply:
@msevestre, @pchelle Btw, _vdiffr_ is what you could be using to test _tlf_ (cf. https://github.com/Open-Systems-Pharmacology/TLF-Library/issues/92), just like in this PR.